content/get-started/quickstart/fork-a-repo.md

```diff
@@ -148,7 +148,16 @@ Fork a repository to start contributing to a project. {% data reusables.reposito
 
 ## Celebrate
 
-You have now forked a repository, practiced cloning your fork, and configured an upstream repository. What do you want to do next?
+You have now forked a repository, practiced cloning your fork, and configured an upstream repository. For more information about cloning the fork and syncing the changes in a forked repository from your computer see "[Set up Git](/articles/set-up-git)".
+
+You can also create a new repository where you can put all your projects and share the code on {% data variables.product.prodname_dotcom %}.
+
+For more information see, "[Create a repository](/articles/create-a-repo)".
+
+Each repository has its own identity on {% data variables.product.prodname_dotcom %} that is associated with some people, or an organization. You can interact with people, repository on {% data variables.product.prodname_dotcom %}. for more information see, "[Be social](/articles/be-social)".
+
+You can also {% data reusables.support.connect-in-the-forum-bootcamp %} and ask your queries. 
```

Comment:
This is difficult. You've used a "reusable" which is good for making articles consistent and easier to maintain. 😄 

However, this reusable was written as a bullet point - so it starts with a capital letter which makes it unsuitable for use within a sentence. 😞 

I've checked where this is used, and it is used only in these four articles that you're editing. So here I think the best solution would be:

1. Edit the text of the reusable so that it matches what you want to say in all of these articles. It is stored here: https://github.com/github/docs/blob/main/data/reusables/support/connect-in-the-forum-bootcamp.md. I'd recommend something like this, which is based on one of the articles above:
    `{% data variables.product.product_name %} has a great support community where you can ask for help and talk to people from around the world. Join the conversation on [Github Support Community](https://github.community/).`
2. Replace the current text about the support community in each of these articles with just `{% data reusables.support.connect-in-the-forum-bootcamp %}`.

Reply:
I actually missed that point. However, these reusable are quite handy in terms of adding into the context.